test/stampit-specs.js

```diff
@@ -443,12 +515,12 @@ test('stampit.convertConstructor() composed', function () {
     'Prototype chain function should be mixed in.');
 });
 
-module('State safety');
+module('Props safety');
 
-test('Stamp state deep cloned for object created', function () {
+test('Stamp props deep cloned for object created', function () {
   var deep = { foo: 'foo', bar: 'bar' };
-  var stamp1 = stampit().state({ deep: deep, foo: 'foo' });
-  var stamp2 = stampit(null, { deep: deep, foo: 'foo' });
+  var stamp1 = stampit().props({ deep: deep, foo: 'foo' });
+  var stamp2 = stampit(null, null, null, { deep: deep, foo: 'foo' });
```

Comment:
Signature is `stampit(methods, refs, enclose, props)`. You have `props` and `refs` swapped in the tests.